identity-account/src/account/account.rs

```diff
@@ -134,6 +134,10 @@ impl Account {
     self.storage.as_ref()
   }
 
+  pub fn storage_arc(&self) -> Arc<dyn Storage> {
+    Arc::clone(&self.storage)
+  }
+
```

Comment:
Just call this `storage`. Is this completely necessary by the way?

Reply:
Discussed this, it is necessary. I do wonder if `storage` can instead return `&Arc<dyn Storage>` though.

Reply:
Good suggestion, I think we should try returning `&Arc<dyn Storage>` from `Account::storage` and get rid of `storage_arc`.